src/libinjector/libinjector.h

```diff
@@ -245,7 +245,8 @@ bool inject_function_call(
     x86_registers_t* regs,
     struct argument args[],
     int nb_args,
-    addr_t function_addr);
+    addr_t function_addr,
+    uint64_t* stack_marker);
```

Comment:
Looks like `NOEXCEPT` is missing from this function, might as well add it now that you are modifying it.